presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -651,15 +712,32 @@ private static void assertFileContentsPresto(
             Format format,
             boolean isHiveWriter,
             boolean useSelectiveOrcReader,
-            List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters)
-            throws IOException
+            List<Map<Integer, Map<Subfield, TupleDomainFilter>>> filters,
+            Optional<List<List<Integer>>> desiredOrders)
```

Comment:
I'd drop Optional and use empty list to indicate "no particular order"

Reply:
Oh, why? And how far should I push this change? There is a stack of multiple functions that all call each other with a similar type structure